src/app/compose/compose.component.scss

```diff
@@ -3,3 +3,75 @@
     border: 1px solid #949494;
     padding: 10px;
 }
+
+.draft-card {
+    margin-bottom: 20px;
+}
+
+.draft-title {
+    flex-grow: 1;
+    padding-left: 15px;
+    font-size: 24px;
+}
+
+.draft-actions {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: flex-end;
+    margin-right: 5px;
+}
+
+.draftPreview {
+    max-height: 100px;
+    overflow: hidden;
+
+    span {
```

Comment:
Huh, I didn't know CSS can do nested blocks. Or is it just scss?

Reply:
That's SCSS, pure CSS can't do that unfortunately :(